cclib/parser/orcaparser.py

```diff
@@ -1507,6 +1507,94 @@ def energy_intensity(line):
                 anisotropic.append(float(aniso))
 
             self.set_attribute('nmrtensors', nmrtensors)
+            
+        # -----------------------------------------------------------
+        #  NUCLEUS A = C    0 NUCLEUS B = C    1 
+        #  ( 13C  gnA =  1.405  13C  gnB =  1.405) r(AB) =     2.8677
+        # -----------------------------------------------------------
+        # 
+        # Diamagnetic contribution (Hz):
+        #         0.4891        -0.1270       -0.0000
+        #        -0.1270        -0.2550        0.0000
+        #        -0.0000         0.0000       -0.1388
+        # Paramagnetic contribution (Hz):
+        #         1.1869         0.2802        0.0000
+        #         0.2802        -0.5515        0.0000
+        #        -0.0000         0.0000       -0.0408
+        # Fermi-contact contribution (Hz):
+        #         7.4196         0.0000        0.0000
+        #         0.0000         7.4196        0.0000
+        #         0.0000         0.0000        7.4196
+        # Spin-dipolar contribution (Hz):
+        #         0.7215         0.0394       -0.0000
+        #         0.0394         1.0985       -0.0000
+        #         0.0000         0.0000        3.6092
+        # Spin-dipolar/Fermi contact cross term contribution (Hz):
+        #         1.9743         0.0164        0.0000
+        #         0.0164         2.2237       -0.0000
+        #         0.0000        -0.0000       -4.1983
+        # 
+        # Total spin-spin coupling tensor  (Hz):
+        #        11.7914         0.2090       -0.0000
+        #         0.2090         9.9353       -0.0000
+        #         0.0000         0.0000        6.6509
+        # 
+        #  Diagonalized sT*s matrix:
+        #  
+        #  ssDSO           -0.139           -0.218            0.452  iso=       0.032
+        #  ssPSO           -0.041           -0.592            1.227  iso=       0.198
+        #  ssFC             7.420            7.420            7.420  iso=       7.420
+        #  ssSD             3.609            1.085            0.735  iso=       1.810
+        #  ssSD/FC         -4.198            2.217            1.981  iso=      -0.000
+        #         ---------------  ---------------  ---------------  ----------------
+        #  Total            6.651            9.912           11.815  iso=       9.459
```

Comment:
At the risk of waffling I think we do want the final number, since most people will only care about that and shouldn't have to write NumPy when it's printed right there.